lib/clearance/engine.rb

```diff
@@ -2,6 +2,21 @@
 require "rails"
 
 module Clearance
+  # Makes Clearance behavior available to Rails apps on initialization. By using
+  # a Rails Engine rather than a Railtie, Clearance can automatically expose its
```

Comment:
What does "automatically expose its routes" mean in this context? That we don't have to manually load them in the routes file?